examples/lalrpop-app/json.lalrpop

```diff
@@ -0,0 +1,52 @@
+grammar;
+
+// https://datatracker.ietf.org/doc/html/rfc7159
+
+pub Object: () = {
```

Comment:
Am I understanding this correctly that this is parsing the json but then discarding the result rather than building it up into a `Value` type?

Reply:
That's correct.  I did that to try to isolate the parser performance.  Looking through a few of the other examples, I see that they do seem to actually build a Value.  I take it that's the preferred approach here?  I'm happy to have it build a Value if you want.

Reply:
Yes, I'd like the comparisons to be more one-to-one

Reply:
Okay, I've updated to build and output a value.